frontend/src/metabase/entities/containers/EntityCopyModal.tsx

```diff
@@ -14,7 +17,7 @@ interface EntityCopyModalProps {
   onClose: () => void;
   onSaved: (newEntityObject?: any) => void;
   overwriteOnInitialValuesChange?: boolean;
-  onValuesChange?: (values: Record<string, unknown>) => void;
+  onValuesChange?: (values: CopyDashboardFormProperties) => void;
```

Comment:
this is used here https://github.com/metabase/metabase/blob/fd8a4fe8a2e8f7f0d8e3d4d93872e4bb8224dba0/frontend/src/metabase/dashboard/containers/CopyDashboardForm.tsx#L54

So I correct the type.